notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -0,0 +1,340 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Single Rule tests\n",
+    "single_rule = ts_client.get_sigma_rule(rule_uuid='5266a592-b793-11ea-b3de-0242ac130004')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "single_rule.__dict__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "single_rule.id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "single_rule.rule_uuid\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "single_rule.es_query"
+   ]
+  },
+  {
+   "source": [
+    "# Text Sigma examples"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MOCK_SIGMA_RULE = \"\"\"\n",
+    "title: Suspicious Installation of Zenmap\n",
+    "id: 5266a592-b793-11ea-b3de-0242ac130004\n",
+    "description: Detects suspicious installation of Zenmap\n",
+    "references:\n",
+    "    - https://rmusser.net/docs/ATT&CK-Stuff/ATT&CK/Discovery.html\n",
+    "author: Alexander Jaeger\n",
+    "date: 2020/06/26\n",
+    "modified: 2021/01/01\n",
+    "logsource:\n",
+    "    product: linux\n",
+    "    service: shell\n",
+    "detection:\n",
+    "    keywords:\n",
+    "        # Generic suspicious commands\n",
+    "        - '*apt-get install zmap*'\n",
+    "    condition: keywords\n",
+    "falsepositives:\n",
+    "    - Unknown\n",
+    "level: high\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rule_by_text = ts_client.get_sigma_rule_by_text(MOCK_SIGMA_RULE)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rule_by_text.es_query"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rule_by_text.references"
+   ]
+  },
+  {
+   "source": [
+    "# Analyzer"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "timelines = sketch.list_timelines()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "timelines[0].name #sigma_events.jsonl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = sketch.run_analyzer(timeline_name='sigma_events.jsonl', analyzer_name='sigma', ignore_previous=True)"
```

Comment:
also, this is not how you run the analyzer, you will get a deprecation warning printed out, and soon this will be removed... better to do

```python
result = timeline.run_analyzer(analyzer_name='sigma', ignore_previous=True)
```

Reply:
oh that makes it easier then cool